packages/browser/src/tracing/browserTracingIntegration.ts

```diff
@@ -371,6 +396,10 @@ export const browserTracingIntegration = ((_options: Partial<BrowserTracingOptio
         startTrackingInteractions();
       }
 
+      if (detectRedirects && optionalWindowDocument) {
+        addEventListener('click', () => (lastClickTimestamp = timestampInSeconds()), { capture: true, passive: true });
```

Comment:
are there other events, such as key presses, that could indicate a user manually navigating?

Reply:
I guess keypress might also be a good candidate, agreed.

Reply:
👍 also looking at keypress